jvm-agent/src/main/org/jetbrains/kotlinx/lincheck/transformation/Metainformation.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.kotlinx.lincheck.transformation
+
+import org.jetbrains.kotlinx.lincheck.SMAPInfo
+
+internal data class ClassMetaInfo (
```

Comment:
1. Please document. 
2. Why is this `ClassMetaInfo` and not simply `ClassInformation`?